### PR TITLE
fix: date issue when enbling time feild

### PR DESCRIPTION
**Pull request by Ahad-patel**
closes : https://github.com/AppFlowy-IO/AppFlowy/issues/5219

https://github.com/user-attachments/assets/29085f2f-6232-4be6-8d38-54931b24cbaa

### Feature Preview

I have fixed the the date set to 00:00 when the enabling the include time button. Since we are already having the time in the variable "parsedDate", and it was passed withoutTime, I removed this so whenever the time is enabled it will show same time as the time field in the DateTime Picker dialog   

---

#### PR Checklist

- [x] My code adheres to [AppFlowy's Conventions](https://docs.appflowy.io/docs/documentation/software-contributions/conventions)
- [x] I've listed at least one issue that this PR fixes in the description above.
- [x] I've added a test(s) to validate changes in this PR, or this PR only contains semantic changes.
- [x] All existing tests are passing.

### CODE REVIEW

**Comment by LucasXu0** [comment]
@Ahad-patel Can you include a necessary integration test for it? You can refer to this file: https://github.com/AppFlowy-IO/AppFlowy/blob/23e3650570e6cedad784b729786edbf525fec4ea/frontend/appflowy_flutter/integration_test/desktop/document/document_with_inline_page_test.dart#L4

Let me know if you have any questions when implementing it.

**Comment by Ahad-patel** [comment]
@LucasXu0 ok, I'll work on that

**Comment by Ahad-patel** [comment]
@LucasXu0 I have added integration tests for date with time and reminder document field. Please let me know if any changes needed.

**Comment by Ahad-patel** [comment]
Hey @LucasXu0, Flutter-CI / prepare-windows (windows-latest) (pull_request) is failing, can you help me resolve this?

**Comment by LucasXu0** [comment]
@Ahad-patel Merging the latest main branch will fix it. It's a bug in vcpkg.

File: frontend/appflowy_flutter/integration_test/desktop/document/document_with_date_reminder_test.dart [hunk 1]
```diff
@@ -0,0 +1,125 @@
+import 'package:appflowy/generated/flowy_svgs.g.dart';
```
Comment (LucasXu0) [inline frontend/appflowy_flutter/integration_test/desktop/document/document_with_date_reminder_test.dart:1]: Please import this test file in `document_test_runner.dart`.

**Comment by Ahad-patel** [comment]
@LucasXu0 I have added `document_with_date_reminder_test.dart` file to `document_test_runner.dart` and merged main with my current branch. Please let me know if any more changes needed.

**Pull request merged**